src/joint.dia.cell.js

```diff
@@ -504,7 +518,25 @@ joint.dia.Cell = Backbone.Model.extend({
     removeAttr: function(path, opt) {
 
         if (_.isArray(path)) {
-            _.each(path, function(p) { this.removeAttr(p, opt); }, this);
+
+            var clone = path.slice();
+            clone.unshift('attrs');
+            return this.removeProp(clone);
+        }
+
+        return this.removeProp('attrs/' + path, opt);
+    },
+
+    removeAttrs: function(path, opt) {
```

Comment:
The method supposed to remove multiple attributes i.e. `removeAttrs(paths, opt)`.  If `paths` is a string do not remove single attribute. That's why `removeAttr()` is there.